Dockerfile

```diff
@@ -5,22 +5,33 @@ FROM node:16.19.0 as build
 
 RUN mkdir -p /base/cache
 ENV YARN_CACHE_FOLDER=/base/cache
+ENV NODE_ENV=production
```

Comment:
oops, removing this from build stage, this prevents `ui` to install webpack and all of its dev-dependencies.
Other parts are already using `--production` so we can safely leave this env var in the last stage only

Reply:
ok, turns out having NODE_ENV=production breaks graphql endpoints.. no idea why yet. Will remove that also